packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -32,6 +34,51 @@ export interface TableProps {
     streamSpecification?: StreamViewType;
 }
 
+export interface DynamoDBAutoScalingProps {
+    /* tslint:disable:max-line-length */
+    /**
+     * The minimum value that Application Auto Scaling can use to scale a target during a scaling activity.
+     * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-applicationautoscaling-scalabletarget.html#cfn-applicationautoscaling-scalabletarget-mincapacity
+     */
+    /* tslint:enable:max-line-length */
+    minimumCapacity: number;
+    /* tslint:disable:max-line-length */
+    /**
+     * The maximum value that Application Auto Scaling can use to scale a target during a scaling activity.
+     * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-applicationautoscaling-scalabletarget.html#cfn-applicationautoscaling-scalabletarget-maxcapacity
+     */
+    /* tslint:enable:max-line-length */
+    maximumCapacity: number;
+    /* tslint:disable:max-line-length */
+    /**
+     * The target value for the metric.
```

Comment:
"Application Auto Scaling ensures that the ratio of consumed capacity to provisioned capacity stays at or near this value. You define TargetValue as a percentage." 

I know the regular docs use the more terse explanation, but they don't provide much value. I think this explanation from [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.CLI.html) is much better

Reply:
Makes sense. Will update accordingly.